Code/GraphMol/MolAlign/Wrap/rdMolAlign.cpp

```diff
@@ -197,6 +198,39 @@ double AlignMolecule(ROMol &prbMol, const ROMol &refMol, int prbCid = -1,
   return rmsd;
 }
 
+double getConformerRMS(ROMol &mol, int confId1, int confId2,
+                       python::object atomIds = python::list(),
+                       bool prealigned = false) {
+  double rmsd;
+  if (atomIds) {
+    std::vector<unsigned int> *aIds = _translateIds(atomIds);
+    rmsd = MolAlign::getConformerRMS(mol, confId1, confId2, aIds, prealigned);
+    delete aIds;
+  } else {
+    rmsd = MolAlign::getConformerRMS(mol, confId1, confId2, 0, prealigned);
+  }
+  return rmsd;
+}
+
+double getBestRMS(const ROMol &ref, ROMol &probe, int refConfId,
+                  int probeConfId, python::object maps = python::list()) {
+  double rmsd;
+  if (maps) {
+    std::vector<MatchVectType> atomMaps;
+    //    BOOST_FOREACH (const python::object &map, *maps) {
+    //      MatchVectType *atomMap = _translateAtomMap(map);
+    //      atomMaps.push_back(*atomMap);
+    //    }
```

Comment:
How do I convert the maps (list of atom mappings = list of list) to a vector of atomMaps? What I have commented here doesn't work.